packages/cloud_firestore/cloud_firestore/lib/src/firestore.dart

```diff
@@ -315,6 +315,7 @@ class FirebaseFirestore extends FirebasePluginPlatform {
       persistenceEnabled: settings.persistenceEnabled,
       host: settings.host,
       cacheSizeBytes: settings.cacheSizeBytes,
+      experimentalForceLongPolling: settings.experimentalForceLongPolling,
```

Comment:
Since it's only available for web, I would rename it `webExperimentalForceLongPolling`

Reply:
Resolved